dts/bindings/serial/renesas,rcar-scif.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021, IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+#
+description: Renesas R-Car UART controller
+
+compatible: "renesas,rcar-scif"
+
+include: uart-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: false
```

Comment:
This seems a bit redundant

Reply:
Well,
next step is to introduce interrupt based UART, so that's why we are mentioning  interrupts in the bindings.

Reply:
Fixed thanks !